plan/new_test.go

```diff
@@ -3,45 +3,186 @@ package plan
 import (
 	"testing"
 
+	"github.com/bradfitz/slice"
```

Comment:
do you really need this dependency? Avoid adding dependencies that save a couple of lines of code. You can very easily create a custom type and use the standard sort package.